### DISCUSSION TITLE
Thanks for the Phono Filter design

### DISCUSSION THREAD

**Discussion by kode54** [RobinSchmidt/RS-MET#340] [Show and tell]
I needed to cobble together a quick process to apply de-emphasis to a recording I accidentally made with the phono pre-amp disabled on my turntable. But then, maybe it was beneficial that I did the cleanup on such a recording before doing a de-emphasis on it afterward.

I may have something to show off, but it's mostly just nostalgic happy content that may or may not be valid to share, since it's an old K-tel record from 1981.

**Comment by RobinSchmidt** [comment]
Thanks. I made that filter a long time ago as part of a vinyl-scratching emulation algorithm which was supposed to go into a DJ software for a client.
